atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -416,7 +417,13 @@ def __init__(self, data, pulp_data, odcs_config, koji_tag=None, arches=None):
         self.arches = arches or []
         self.multilib_arches = []
         self.multilib_method = None
-        self.modular_tags = data.get('modular_koji_tags', [])
+        self.modular_tags = data.get('modular_koji_tags')
+        self.koji_tag = koji_tag
+
+        if self.modular_tags is True:
+            if not self.koji_tag:
+                raise ValueError('koji_tag is required when modular_koji_tags is True')
```

Comment:
Could we say that a koji target is required (this error msg will be displayed to the user, and `koji_tag` won't mean much in this sense)

Reply:
well the same message is in validation

thing is, there could be koji target, without tag I'd say soooo

Reply:
also this can happen only when using osbs-client directly